### PR TITLE
add kubelet version and OS image version in node/worker pool overview dashboard

### PR DESCRIPTION
**Pull request by tedteng**
**How to categorize this PR?**

/area ops-productivity
/kind enhancement

**What this PR does / why we need it**:
add two graphs showing kubelet version and OS image version in node/worker pool overview dashboard. It helps dod/ the end users check version upgrades during the rolling update or the version status of nodes.

image
image

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
showing kubelet version and OS image version in Plutono Node/Worker Pool overview dashboard
```

### CODE REVIEW

**Comment by tedteng** [comment]
ping @istvanballok @rickardsjp

File: pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json [hunk 1]
```diff
@@ -627,6 +626,200 @@
...
+          "interval": "",
+          "legendFormat": "{{os_image}}/{{label_worker_gardener_cloud_pool}}",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "OS Image of Nodes in Worker Group(s) All",
```
Comment (istvanballok) [inline pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json]:
```suggestion
      "title": "OS Image of Nodes in Worker Group(s) $worker_group",
```

File: pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json [hunk 2]
```diff
@@ -627,6 +626,200 @@
...
+          "interval": "",
+          "legendFormat": "{{kubelet_version}}/{{label_worker_gardener_cloud_pool}}",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Kubelet versions of nodes in Worker Group(s) All",
```
Comment (istvanballok) [inline pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json]:
```suggestion
      "title": "Kubelet versions of nodes in Worker Group(s) $worker_group",
```

File: pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json [hunk 3]
```diff
@@ -1090,4 +1283,4 @@
   "title": "Node/Worker Pool Overview",
   "uid": "node-worker-pool-overview",
   "version": 1
-}
+}
```
Comment (istvanballok) [inline pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json]: maybe add a new line

**Review by istvanballok — CHANGES_REQUESTED** [review]
There is a minor typo that the hard coded "All" value is used in the panel titles instead of the variable "worker_group", like for the other panels.

The release notes sentence:
```
showing kubelet version and OS image version in Prometheus Node/Worker Pool overview dashboard
```

could be slightly improved: Prometheus ... dashboard -> Plutono ... dashboard.

Otherwise, it looks good to me.

File: pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json (hunk 1 above)
Comment (tedteng) [reply pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json]: 7c78ef96a

File: pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json (hunk 2 above)
Comment (tedteng) [reply pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json]: 7c78ef96a

File: pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json (hunk 3 above)
Comment (tedteng) [reply pkg/component/plutono/dashboards/shoot/owners/worker/node-pool-dashboard.json]: 7c78ef96a

**Review by istvanballok — APPROVED** [review]
/lgtm

**Pull request merged**